packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -174,7 +175,8 @@ const tlsVersionTest = async (_, res) => {
  * Express handler that enables the cache and returns a JSON response with diagnostic values.
  */
 const cacheTest = async (req, res) => {
-    res.set('Cache-Control', 's-maxage=60')
+    let duration = req.params.duration || '60'
+    res.set('Cache-Control', `s-maxage=${duration}`)
```

Comment:
a little input validation would stop the user from being able to add in arbitrary strings to the end of the cache control header, see https://github.com/SalesforceCommerceCloud/pwa-kit/compare/add-reference-app-testing-endpoints#diff-b58817825fcbdf4737f480cad9dfd997560c0cd95ccc6977f3256d813079de27R184 :)

Reply:
thanks for pointing out the router level validation!